documentation/content-editors/api-guidance-pages.md

```diff
@@ -0,0 +1,97 @@
+---
+title: API Guidance Pages
+---
+
+This document is designed to assist content editors with making changes to the API guidance pages. The general steps to make changes to API guidance are:
+
+- [API Guidance Pages](#main-content)
+  - [Start a Github Codespace](#start-a-github-codespace)
+  - [Navigate to/create the Markdown file for the guidance page](#navigate-to-create-the-markdown-file-for-the-guidance-page)
+  - [Construct the page/contents using supported Markdown syntax](#construct-the-page-contents-using-supported-markdown-syntax)
+  - [Preview your changes](#preview-your-changes)
+  - [Raise a pull request](#raise-a-pull-request)
+  - [Seek approval/merge your changes](#seek-approval-merge-your-changes)
+
+## Start a Github Codespace
+
+[GitHub Codespaces](https://github.com/features/codespaces) enable you to make changes to the application/guidance pages in a pre-configured environment from within your browser. To use Codespaces you will need:
+
+- A GitHub account (via a tools request in #digital-tools-support on Slack)
+- To be part of the DFE-Digital GitHub organization (go to [your organisations list](https://github.com/settings/organizations) and look for DFE-Digital).
+- Be granted access to the [RECT public repository](https://github.com/DFE-Digital/register-early-career-teachers-public).
+- Have Codespaces enabled for your account.
+
+Start by creating a branch to work from. You can do this by navigating to the [branches section in the GitHub repository](https://github.com/DFE-Digital/register-early-career-teachers-public/branches) and clicking `New branch`. Here, enter a branch name to describe your changes and click `Create new branch`:
+
+<a href="../images/new-branch.png" target="blank"><img src="../images/new-branch.png" width="300"/></a>
+
+Now we can switch to our branch by navigating to the [main repository page](https://github.com/DFE-Digital/register-early-career-teachers-public) and selecting the branches drop-down, searching for and choosing our branch:
+
+<a href="../images/select-branch.png" target="blank"><img src="../images/select-branch.png" width="300"/></a>
+
+Finally, we want to start a Codespace on this branch by clicking `Code` -> `Create codespace on <your branch name>` (you should see "Codespace usage for this repository is paid for by DFE-Digital." in the dialog):
+
+<a href="../images/create-codespace.png" target="blank"><img src="../images/create-codespace.png" width="300"/></a>
+
+The first time you create a Codespace it can take quite a while to spin up.
+
+Once Codepsace is ready click on the `TERMINAL` tab and enter command `./bin/dev` to start rails application.
```

Comment:
```suggestion
Once Codespace is ready click on the `TERMINAL` tab and enter command `./bin/dev` to start rails application.
```

Reply:
I added why because I wasn't sure and asked Pete, I'd never done this before. Learn something new everyday :D